moped-editor/src/views/projects/projectView/ProjectComponents/ComponentListItem.js

```diff
@@ -65,10 +84,7 @@ export default function ComponentListItem({
       }}
     >
       <ListItem dense button onClick={onListItemClick}>
-        <ListItemText
-          primary={listItemPrimaryText}
-          secondary={component.moped_components?.component_subtype}
-        />
+        <ListItemText {...listItemText} />
```

Comment:
What do you think about keeping the `ListItem`s the same height regardless of whether there is secondary text or not? I don't see a way to do this without CSS so this could probably go into consideration with the style refinements that you mentioned.

Reply:
Moved towards this in https://github.com/cityofaustin/atd-moped/pull/982